src/components/CellLineTable/DiseaseTableColumns.tsx

```diff
@@ -1,31 +1,53 @@
 import React from "react";
 import { Flex } from "antd";
 import classNames from "classnames";
+import { Link } from "gatsby";
 
 import {
     Clone,
     UnpackedDiseaseCellLine,
     UnpackedGene,
     UnpackedNormalCellLine,
+    CellLineStatus,
 } from "../../component-queries/types";
 import { formatCellLineId, getCloneSummary } from "../../utils";
 import GeneDisplay from "../GeneDisplay";
 import ParentalLineModal from "../ParentalLineModal";
 import CloneSummary from "../CloneSummary";
 
 import {
-    cellLineIdColumn,
     certificateOfAnalysisColumn,
     obtainLineColumn,
 } from "./SharedColumns";
-import { smBreakPoint, mdBreakpoint, CellLineColumns } from "./types";
+import { smBreakPoint, mdBreakpoint, CellLineColumns, UnpackedCellLine } from "./types";
 
 const {
     clones,
     lastColumn,
     snpColumn,
+    cellLineId,
 } = require("../../style/table.module.css");
 
+const cellLineIdColumn = {
```

Comment:
nit, but there is not need to define this separately and then make a copy of it below. you can just define it right in the array